com.microsoft.mrtk.input/Simulation/Devices/ControllerSimulationSettings.cs

```diff
@@ -60,17 +61,57 @@ public InputActionReference ChangeNeutralPose
 
         [SerializeField]
         [Tooltip("The initial pose of the simulated hand")]
-        private SimulatedHandPose defaultPose = SimulatedHandPose.Neutral;
```

Comment:
Is there any way for some sort of migration path here? Even the defaults don't match 1 to 1 (`SimulatedHandPose.Neutral` is 0 while `GestureId.Open` is 2), so if I've created a custom settings asset I'm now likely broken.

Reply:
I think I'm willing to put in a breaking change here due to how niche the original enum was, and how it was an redundant enum with the pre-existing GestureIds :(

Reply:
Yeah, definitely happy to see a reduction in redundant enums!